server/src/test/java/org/elasticsearch/cluster/coordination/CoordinatorTests.java

```diff
@@ -359,9 +286,19 @@ public void onFailure(String source, Exception e) {
         }
     }
 
-    @SuppressWarnings("unchecked")
-    private static void processMessageReceived(TransportRequest request, RequestHandlerRegistry requestHandler,
-                                               TransportChannel transportChannel) throws Exception {
-        requestHandler.processMessageReceived(request, transportChannel);
+    private static Runnable onNode(Runnable runnable, DiscoveryNode node) {
```

Comment:
I meant to put the `DiscoveryNode` thing first, otherwise it ends up a long way from the function call if the `Runnable` is an anonymous class.

Reply:
ok. With the recent changes, there were no longer any anonymous classes. I've still changed the order